TASagentTwitchBot.Core/Core/Cryptography.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace TASagentTwitchBot.Core.Core
```

Comment:
`TASagentTwitchBot.Core.Core` 🤮 

Haha, this naming snafu came from quirks of history, related to how I'd originally set up the project.  `TASagentTwitchBot.Core` is sufficient.